pkg/controller/daemonset/daemonset_controller.go

```diff
@@ -805,7 +799,9 @@ func (dsc *ReconcileDaemonSet) podsShouldBeOnNode(
 	nodeToDaemonPods map[string][]*corev1.Pod,
 	ds *appsv1alpha1.DaemonSet,
 	hash string,
-) (result reconcile.Result, nodesNeedingDaemonPods, podsToDelete []string, err error) {
+) (duration time.Duration, nodesNeedingDaemonPods, podsToDelete []string, err error) {
+
+	var requeueDuration = requeueduration.Duration{}
```

Comment:
Why do you use requeueduration.Duration here? I don't see concurrent access issue in this method.

Reply:
> Why do you use requeueduration.Duration here? I don't see concurrent access issue in this method.

I just find it convenient to update time duarion with requeueduration.Duration,  i will fix it.